src/main/java/core/basesyntax/file/FileWriterImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.file;
+
+import exception.FileException;
```

Comment:
Ensure that the `FileException` class is correctly defined and imported. If it is not present in the project, you need to create it to handle file-related exceptions properly.